pyproject.toml

```diff
@@ -90,7 +90,7 @@ docs = [
 ]
 
 ci_tox = [
-    "tox==4.25.0",
+    "tox-uv>=1.26.2",
```

Comment:
There is `"tox-uv==1.26.0"` entry in `dev` target, the requirements look a bit messy. Is it possible to avoid overlapping between the targets?